packages/components/src/color-picker/style.scss

```diff
@@ -28,6 +28,10 @@
 .components-color-picker {
 	width: 100%;
 	overflow: hidden;
+
+	* {
+		box-sizing: border-box;
```

Comment:
Do you think we can target specific elements instead of using `*`?

Reply:
What I was going for is applying `box-sizing: border-box;` to all the elements within the component. 

Targeting each specific item would be cumbersome and difficult to maintain, right?

Reply:
I'd say it depends the number of these elements and whether we control them or not. It's a minor comment anyway :)

Reply:
I think this is a safe approach to normalize styles for components that appear in the context of `ColorPicker`. All of the components that render within are from the `@wordpress/components`, and the current styles of the entire lib depends on a global `box-sizing: border-box;` anyway 👍 .

Reply:
Thanks for the feedback @youknowriad and @ItsJonQ!